src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -120,6 +122,39 @@ const LoginForm: React.FunctionComponent<PropsType> = ({
             <span className="input-error">Password is required</span>
           )}
         </div>
+        <div className="input-group">
```

Comment:
We can use the new `DateOfBirthField` and `TicketCodeField` components instead of this now can't we?